local_test2.py

```diff
@@ -0,0 +1,65 @@
+
```

Comment:
Please exclude this file

Reply:
Why? we should have a reference to local test execution (some customers are using it )
Don't forget that this is an "open source" tool, customers can contribute and build their own bots, and for that, it's good to have local test execution reference

Reply:
Ok i see, so if you think this file should be included add comment explaining how to use it, change it name (remove the 2) and remove private data